src/components/token/TokenTable/index.tsx

```diff
@@ -138,11 +138,11 @@ const HeaderTitle = styled.span`
 
 const TokenWrapper = styled.div`
   display: flex;
-  align-items: center;
   a {
-    display: none;
+    width: 100px;
   }
   img {
+    margin-right: 10px;
```

Comment:
margin-right: 1rem;

hey @matextrem, we should use rem units instead of px. Thanks! :)

Reply:
Gotcha! Will update this